.travis.yml

```diff
@@ -14,16 +13,15 @@ before_install:
   - conda update --yes conda
   - conda info -a
 install:
-  - wget -q https://raw.githubusercontent.com/qiime2/environment-files/master/latest/staging/qiime2-latest-py36-linux-conda.yml
-  - conda env create -q -n test-env --file qiime2-latest-py36-linux-conda.yml
+  - conda env create -q -n test-env --file resources/environment.yml
```

Comment:
If this line is successful, lines 17-24 should be obsolete

Reply:
yup, there are already removed.

Reply:
Latest diff shows they are still there. To be clear I am referring to these lines:
``` 
  - conda install --yes -n test-env --file ci/conda_requirements.txt -c biocore
  - conda install --yes -n test-env cython
  - source activate test-env  
  - pip install -r ci/pip_requirements.txt
  - pip install -e .
  - pip install flake8
  - pip install nose
  - pip install coveralls
```

Reply:
oh no those are still req. will fail without them.

Reply:
Sorry, I think just the following need to be deleted:
```
  - conda install --yes -n test-env --file ci/conda_requirements.txt -c biocore
  - pip install -r ci/pip_requirements.txt
  - pip install -e .
  - pip install flake8
  - pip install nose
  - pip install coveralls
```
Because these should be covered by the resources/environment.yml ? We essentially want the travis environment to be set up the same way as our conda environment when running locally, right?

Leaving:
```
conda install --yes -n test-env cython
source activate test-env

```

Reply:
Ah I see now yes but we would still need 

``` 
pip install -e .
``` 
so the tests can import TesasSemble. so in total only kept is:

``` 
conda install --yes -n test-env cython
source activate test-env
pip install -e .
```

Reply:
yup, that passes.

Reply:
all the ci reqs. that were necessary are in the yml.

Reply:
really we don't need ```pip install -e .``` _now_ but as soon as the first real test is run we will need it so might as well add it now.